### DISCUSSION TITLE
Consiglio modem per Home Assistant?

### DISCUSSION THREAD

**Discussion by ElyFNV** [indomus/forum#1982] [HUB: Home Assistant]
Ciao a tutti!
Da ormai 8 mesi mi sono trasferito in una nuova casa, dove ho scelto Fastweb come operatore (per prezzi convenienti ed ottima velocità di navigazione). Sto utilizzando quindi il modem proprietario FastGate. Tuttavia, ormai da 1 mese a questa parte ogni poco vado incontro a continue disconnessioni, rallentamenti della rete e veri e propri blocchi della rete internet fissa. Improvvisamente e randomicamente la velocità internet, da 91mb/s scende a 2mb/s, poi ogni dispositivo smart che ho in casa si disconnette e non si riconnette finché non riavvio manualmente (spegnendo e riaccendendo il fastgate).
Questa cosa sta diventando sempre più faticosa da sopportare: non ho più una smart home ma una stupid home!!!
Ho provato a contattare fastweb ma ovviamente la loro risposta è come sempre inutile e senza senso: fanno dei "controlli" e non cambia mai nulla.
Volevo sapere se secondo voi fosse magari un problema del modem stesso, che non riesce a sostenere tutti i dispositivi smart che ho collegato... E soprattutto se qualcuno avesse qualche consiglio magari su un modem differente da poter acquistare. Stavo pensando ad un FritzBox, ma ovviamente non saprei su quale modello andare.
Partiamo dal presupposto che io ho una FTTC (in rame quindi). Qualcuno saprebbe cosa consigliarmi? Cerco un modem che sia fatto apposta per sostenere una grande quantità di dispositivi connessi tutti assieme, come accade in una smart home!
Ah! Altra cosa: il modem fastgate non permette di aggiungere più di 15 IP statici (dopo 15 mi dà errore e devo eliminarne alcuni per poterne aggiungere degli altri)... vorrei quindi un modem che permetta di poter dare quanti IP statici voglia (se questa cosa esiste). E che permetta anche di poter effettuare un backup della configurazione del modem così da poterla importare in caso di reset dello stesso (perché ovviamente il FastGate non ha questa possibilità e l'altra sera dopo un reset remoto da parte di Fastweb ho perso tutta la configurazione!!!).
VI ringrazio in anticipo! Aiutatemi per favore!

**Comment by essegi65** [comment]
Premetto che non ho fastweb, ma per il modem tutta la vita marca fritzbox (modem e accessori per ampliare il wifi con rete mesh) da quando l'ho installato mi è cambiata in meglio tutta la gestione wifi e cablato.

**Reply by ElyFNV** [reply]
> Premetto che non ho fastweb, ma per il modem tutta la vita marca fritzbox (modem e accessori per ampliare il wifi con rete mesh) da quando l'ho installato mi è cambiata in meglio tutta la gestione wifi e cablato.

Cercando un po sul web ho trovato alcune discussioni in cui consigliavano il FritzBox 7530 AX o il FritzBox 7590.
Te quale mi consigli? Riguardo alla rete mesh io al momento in casa ho 2 extender collegati tramite cavo ethernet alla rete (perché ho la casa su 2 piani e non mi arrivava la rete wifi al piano sotto). Se acquisto il FritzBox posso continuare comunque ad utilizzare gli extender per estendere la wifi al piano sotto e comandare le lampadine e prese smart del sottosuolo?
Grazie!!

**Reply by essegi65** [reply]
Io ho il 7590 (acquistato 2 anni fa) e mi trovo da dio come detto prima. Per gli extender non saprei io uso sempre l'extender di fritzbox collegato via cavo, direi che forse puoi provare acquistando solo il modem e poi nel caso ampliare. Il wifi di fritz è molto performante io al piano sotto ci arrivo senza extender (quello che uso è per un sottotetto)

**Reply by 50494554524F** [reply]
ci puoi attaccare un mini router da usare come access point, tp.link o xiaomi costano niente...

**Reply by NdK73** [reply]
Confermo la bontà dei prodotti AVM. Anche io ho un 7590 da 4-5 anni, con repeater mesh (due 2400 e un 3000) e un 7530 in un'altra casa (collegati con la loro VPN :) ).
Se non ti serve il supporto DSL, meglio il 7530. Ottima la telefonia, anche se si potrebbe migliorare.

**Reply by indomus** [reply]
Oggi pomeriggio esce la recensione del 7590 AX ;)

**Reply by ElyFNV** [reply]
Allora. Alla fine ho acquisto un FritzBox 7530 AX sfruttando anche le offerte del Prime Day di Amazon. Installato e configurato correttamente (dopo aver litigato con Fastweb per avere il via alla configurazione del modem generico). Internet e fonia perfettamente funzionanti.
Insomma ero contentissimo... sono stato 3 giorni senza problemi. Nessun rallentamento di connessione e nessuna caduta. Fino ad oggi pomeriggio... Improvvisamente è successo esattamente la stessa cosa che mi accadeva con il FastGate: ero connesso col cellulare in Wifi al modem Fritzbox, la connessione è diventata lentissima (1mb/s in dw da 100 che era) e tutti i dispositivi smart, compreso lo stesso HA sono diventati non disponibili. E' rimasto così per circa 5-10 minuti, poi è ripartito tutto.
Quindi unica differenza da quello che mi accadeva col FastGate è che quel modem, quando dava questi problemi di disconnessione, doveva necessariamente essere riavviato manualmente altrimenti non ripartiva (e poi, una volta riavviato dovevo smattare per permettere a tutte le luci di riconnettersi... non so per quale motivo ma finivo sempre con l'accenderle e spegnerle manualmente per forzarle ad una riconnessione). Con il FritzBox questo non è accaduto; nel senso che una volta ripartito ha riconnesso tutti quanti i dispositivi automaticamente.
Tuttavia resto sempre basito per quello che mi sta accadendo: non riesco a capire come mai per 7 mesi non è successo mai nulla e adesso, improvvisamente, ho questi rallentamenti con conseguenti disconnessioni casuali (anche cambiando modem!!!). E poi perchè dopo aver sostituito il FastGate col FritzBox comunque sono riuscito a fare 3 giorni interi senza interruzioni???
Cavolo pensavo davvero di aver risolto una volta per tutte. Non so più che fare... Qualcuno saprebbe consigliarmi?
Grazie a tutti in anticipo!

**Reply by snakuzzo** [reply]
Non so se mi sono perso qualcosa nei precedenti messaggi. Ma questo problema ce l'hai solo sui client collegati in wifi o anche su quelli via ethernet?

**Reply by ElyFNV** [reply]
Ecco questa è l'unica cosa che non sono ancora riuscito a controllare. Perchè ho un solo PC fisso connesso con etherhet e non mi è mai venuto in mente di controllare. Prossima volta che accade lo farò e vi aggiornerò sicuramente!
Ma poi una cosa che non mi torna: io ho tutte luci smart Tuya in casa e le ho integrate tutte quante in LocalTuya in Home Assistant. A volte quando cade internet le luci continuano comunque a funzionare correttamente in locale (dovrebbe teoricamente essere quello lo scopo infatti no?); quando invece mi cade la linea e mi succedono queste "disconnessioni" casuali tutte le luci diventano "NON DISPONIBILI". Mi chiedo... come mai? Non dovrebbero continuare a funzionare anche senza internet essendo in locale? Mi sono perso qualcosa???

**Reply by snakuzzo** [reply]
Ecco il perché della domanda che ti ho fatto poco fa. Può darsi che non sia un problema legato alla tua connessione internet o al modem specifico...ma un qualcosa legato al WIFI

**Reply by ElyFNV** [reply]
Perdona la mia ignoranza... Prossima volta che accade vado immediatamente a vedere se il PC fisso connesso con ethernet subisce disconnessioni o meno e ti faccio sapere!!
Ma se posso chiedere... In caso il pc fisso continui a funzionare cosa pensi che possa significare? Che tipo di problemi legati al wifi potrei avere? Soprattutto così improvvisamente? Qualcosa che per 7 mesi non ho mai avuto e ora ho in continuazione... (nell'ultimo mese poi non ho nemmeno aggiunto nuovi dispositivi smart alla casa quindi boh).... può essere che tutti questi dispositivi wireless sovraccarichino il modem? Perchè parlando su internet sento di persone che hanno domotizzato case di 3 piani piene di dispositivi smart. Io ho una casa di 75 m quadri... e 77 dispositivi smart (tra luci, prese ecc...)
Non so che cosa sto sbagliando sinceramente. Unica cosa che ho è che non possiedo una rete wifi mesh. Ho un modem Fritzbox e 2 repeater (https://www.tp-link.com/it/home-networking/wifi-router/archer-ax10/) che fungono da access point dove il wifi del modem principale non arriva. E questi sono connessi via cavo...

**Reply by snakuzzo** [reply]
Se il pc dovesse andar bene (cosa che credo fortemente), tocca vedere poi con calma andando per esclusione e facendo prove.
Dovresti provare a vedere se lo fanno tutti i client o solo quelli collegati ai repeater...oppure cominciare a smanettare sulle impostazioni del WiFi, verificando innanzi tutto se capita solo sulla wifi a 2.4GHz o anche sulla 5 GHz (e qui ti consiglierei di provare a separare i SSID). Se hai problemi solo sulla 2.4GHz potresti avere interferenze e quindi dovresti provare a giocare cambiando tra i vari canali...
Ad ogni modo, capire quali possano essere i problemi, è difficile da capire (almeno per me).

**Reply by ElyFNV** [reply]
Hai ragione. Già è complicato da qui ahah. Comunque prossima volta che accade farò un po' di prove. Intanto ti ringrazio perché sei stato davvero gentilissimo.
Se riuscirai con calma a seguirmi un minimo quando accadrà ancora ti darò tutte le info che potrebbero essere utili alla possibile indagine sul problema. Insomma quello che voglio dire è che io non mi intendo purtroppo di sistema internet in sé. Diciamo che su questo non sono molto smanettone ecco. Quindi avere qualcuno che un po' di più di me ci capisce mi farebbe sentire più tranquillo ahah.
Posso continuare a scrivere qui prossima volta sperando in una tua risposta?

**Reply by snakuzzo** [reply]
Scrivi qui...certo.
Comunque non ci sono solo io (che magari so solo quattro cavolate in croce). C'è anche altra gente che di sicuro ne sa di più.

**Reply by ElyFNV** [reply]
Sisi quello si certo. Però fa già piacere avere qualcuno che risponde e mi sta dietro.
Grazie!

**Reply by ElyFNV** [reply]
Allora. Dal momento che la rete è caduta 4 o 5 volte dall'ultima volta che ho scritto, sono qui a darti i risultati dei test effettuati.
Partiamo dicendo questo:
1. Nel momento in cui la rete 2.4 relativa al modem principale smette di funzionare, la rete "cablata" continua tranquillamente a funzionare (speed test eseguiti hanno, infatti, evidenziato una velocità in DWL di 91mb/s via ethernet).
2. Anche la rete 5 continua tranquillamente a funzionare (anche qui, uno speed test indicava una velocita di DWL di circa 87 mb/s)

L'altra sera, dopo aver analizzato la situazione canali delle reti con "Wifi Analyzer" ho visto come la rete del modem principale fosse su un canale abbastanza "incasinato" dalla presenza di altre reti dei vicini di casa. Ho anche visto che sul fritzbox era spuntata l'opzione "Impostare automaticamente le impostazioni per il canale radio (consigliata)"; ho pensato che magari fosse questo il problema. Così ho disattivato l'opzione e impostato un canale fisso (l'1, perchè mi sembrava al momento il più libero). E niente... cosa è successo?
Ovviamente stasera la connessione 2.4 ha smesso di funzionare completamente (la 5 andava e via ethernet funzionava sempre tutto), ma a differenza delle volte precedenti, anche dopo 20 minuti non era ancora ripartita. Ho quindi aperto la pagina del fritzbox e ho eseguito una diagnostica, la quale mi ha detto che il canale 1 era troppo pieno di reti e che però, non essendo più attiva l'opzione di ricerca automatica del canale, il modem non poteva scegliere il canale più adatto (era l'opzione che avevo disattivato il giorno prima, perché pensavo fosse la causa delle cadute di rete 2.4). Insomma... ho riattivato l'opzione e come per magia tutto è ripartito alla perfezione.
Ora, dopo tutto questo ambaradan... io mi chiedo.... CHE CAVOLO POSSO FARE?!?
E' sicuramente un problema di Wifi 2.4, dal momento che via ethernet e sul 5 funziona sempre tutto.
Pensavo fosse un problema di canali troppo "affollati" (e probabilmente lo è), ma come mai allora la connessione cade anche se è attiva l'opzione di ricerca automatica di canali?
E' probabile che i modem dei miei vicini di casa facciano interferenza, magari cambino anche loro in continuazione i canali e, nel momento in cui un canale risulta troppo occupato, la wifi 2.4 vada incontro a rallentamenti e tutto riparta solo quando il modem fritzbox trova un canale più libero a cui connettersi?
E se fosse così, significa che non potro mai risolvere il problema? Dovrei andare a spegnere i modem dei miei vicini? E' questa la soluzione? Non ci voglio credere. Perché ricordiamoci che io per 7 mesi non ho avuto problemi!!! E' da un mese che sono nati tutte queste disconnessioni (e anche cambiando modem, passando dal FastGate originale di Fastweb al fritzbox 7530 AX non è cambiato nulla).
Quindi boh... Se qualcuno avesse da darmi qualche consiglio è bene accetto.
Ho anche pensato fossero i repeater che ho in casa a dare noia, per cui ho acquistato 2 repeater mesh fritzbox, cosi da creare una rete mesh in casa (che su internet dicono essere molto più stabile per la domotica). Ma ora mi viene il dubbio che anche questa cosa non risolverà nulla... che tanto se il modem principale continuerà a dare problemi anche la rete mesh avrà problemi di conseguenza.
E non so più che cosa fare. So solo che non riesco più a vivere in una "Smart Home" continuamente in disconnessione e riconnessione.

**Reply by snakuzzo** [reply]
l'unica prova che mi viene in mente ...vai su wifi > canale radio > altre impostazioni  > standard wireless 2.4 e prova a cambiare tra le varie opzioni.

P.S. il "superpoteri" dei mesh dei prodotti AVM...sono una gran cavolata

**Reply by KC-inDomus** [reply]
mesh AVM a me sinora sempre funzionati da dio. ma in questo scenario direi che i problemi sono ben altri

**Reply by ElyFNV** [reply]
Quindi? Se anche col mesh non risolverò che cosa dovrei fare? Vivere in una casa che si disconnette 2 volte al giorno tutti i giorni? Aspettando che per accendere le luci il modem trovi in canale adatto?
È follia. Ha funzionato per 7 mesi senza problematiche. Adesso tutto improvviso... Boh. Esiste un modo per capire cosa potrebbe essere a dare interferenza? Insomma quando accade ci sarà un modo per identificare la sorgente del problema no? O qualcosa per isolarlo... Non so tipo per schermare il modem da queste interferenze. Boh... Non so più che pensare  e non so dove mettere le mani

**Reply by snakuzzo** [reply]
Hai fatto la prova che ti ho detto?
Più di questo, non credo ti si possa dare altri suggerimenti

**Reply by ElyFNV** [reply]
No ancora no che sono a lavoro. Domani provo. Grazie!
Ti farò sapere...

**Reply by ElyFNV** [reply]
Allora. Non sono riuscito a trovare l'opzione che mi hai detto. Comunque ho installato la rete mesh fritzbox e momentaneamente sembra stabile... Farò passare qualche giorno per vedere se tornano le disconnessioni casuali o se ho risolto...
